src/csv.cr

```diff
@@ -136,8 +136,8 @@ class CSV
   # end
   # io.to_s # => "HEADER\none,two\nthree\n"
   # ```
-  def self.build(io : IO)
-    builder = Builder.new(io)
+  def self.build(io : IO, separator : Char = DEFAULT_SEPARATOR, quote_char : Char = DEFAULT_QUOTE_CHAR)
+    builder = Builder.new(io, separator, quote_char)
     yield builder
```

Comment:
Doesn't the one line `yield Builder.new(io, separator, quote_char)` works?

Reply:
This PR is not about that, sorry.

Reply:
Please, let's stop commenting about syntax. Let's focus on semantic. Same line, different line, it's the same behaviour.

Reply:
I'm probably wrong sorry, but `builder =` allocates, right? Avoiding this and directly `yield` can improve performance?

Reply:
No. `Builder.new` allocates heap memory and puts a reference to it on the stack. That piece on the stack is just called `builder`. You can call it whatever you want or don't give it any name - it makes no real difference.